baselines/fed_debug/fed_debug/dataset.py

```diff
@@ -0,0 +1,225 @@
+"""Handle basic dataset creation.
+
+In case of PyTorch it should return dataloaders for your dataset (for both the clients
+and the server). If you are using a custom dataset class, this module is the place to
+define it. If your dataset requires to be downloaded (and this is not done
+automatically -- e.g. as it is the case for many dataset in TorchVision) and
+partitioned, please include all those functions and logic in the
+`dataset_preparation.py` module. You can use all those functions from functions/methods
+defined here of course.
+"""
+
+import logging
```

Comment:
Flower has its own logging. let's better make use of it. You can do 
```
from flwr.common.logger import log
```
Then replace all your `logging.info(....)` with `log(INFO, ...)` -- the same applies to other types of message (e.g. `WARN`, `DEBUG`). And for all files in your baseline.

Reply:
resolved.